olive/data/component/dataloader.py

```diff
@@ -3,14 +3,25 @@
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
 
-from torch.utils.data import DataLoader
+from torch.utils.data import DataLoader, default_collate
 
 from olive.data.registry import Registry
 
 
 @Registry.register_default_dataloader()
 def default_dataloader(dataset, batch_size=1, **kwargs):
-    return DataLoader(dataset, batch_size=batch_size, **kwargs)
+    def ignore_batch_collate_fn(batch):
+        # ignore to batch the data fields for given inputs
+        to_ignore_fields = kwargs.get("ignore_fields") or ["step"]
```

Comment:
hard-coding `["step"]` might not be a good idea here. especially since this is the `default_dataloader` component.